test/src/tests.coffee

```diff
@@ -68,8 +68,7 @@ asyncTest 'Containers and parents', ->
   e = cont2.end
   f = cont1.end
 
-  a.append(b).append(c).append(d)
-   .append(e).append(f)
+  helper.string [a, b, c, d, e, f]
```

Comment:
consider moving this one to modelTests.coffee, along with any other tests that exercise the model without touching the container or view.